lib/Driver/ToolChain.cpp

```diff
@@ -21,12 +21,13 @@
 #include "swift/Driver/Compilation.h"
 #include "swift/Driver/Driver.h"
 #include "swift/Driver/Job.h"
+#include "swift/Option/Options.h"
```

Comment:
This doesn't seem to be used in your diff. Or were we implicitly including it before and now we're not?

Reply:
Great catch! Fixed.